pyc.py

```diff
@@ -69,6 +69,8 @@ def compile_bin_op(ctx: Context, bo: ast.BinOp) -> str:
         # TODO: handle non-longs
         ctx.body.write(f"if (PyLong_Check({l}) && PyLong_Check({r})) {{\n", ctx.indentation)
         ctx.body.write_statement(f"{result} = PyLong_FromLong(PyLong_AsLong({l}) + PyLong_AsLong({r}))", ctx.indentation + 1)
+        ctx.body.write(f"}} else if (PyUnicode_Check({l}) && PyUnicode_Check({r})) {{\n", ctx.indentation)
```

Comment:
I'm doing some refactoring to have a standalone libpyc.c module that has helper functions for common tasks like addition and whatnot. I'll pull in this MR after that refactoring.